packages/api/tests/controllers/user.test.js

```diff
@@ -37,3 +36,58 @@ describe('GET user Endpoint', () => {
     expect(res.statusCode).toBe(400);
   });
 });
+
+describe('POST user Endpoint', () => {
+  const user = {
+    email: 'test@test.com',
+    username: 'test_username',
+  };
+  let newUser;
+
+  beforeEach(async () => {
+    return db.User.create(user);
+  });
+
+  afterEach(async () => {
+    await db.User.destroy({ where: {} });
```

Comment:
Sorry, this is somewhat my fault, but probably worth being consistent here, either using the `async`/await` in both functions or not-`async`/returning the promise.

Also, I think you can just `db.User.destroy();` without the empty `where`.

Reply:
When we remove the  `where` we get the following error.

`Missing where or truncate attribute in the options parameter of model.destroy.`

Reply:
Ah, my mistake then!